packages/uikit/src/components/Image/TokenPairImage.tsx

```diff
@@ -25,6 +27,15 @@ const TokenPairImage: React.FC<React.PropsWithChildren<TokenPairImageProps>> = (
         height={secondaryImageSize}
         {...secondaryImageProps}
       />
+      {chainLogoSrc ? (
+        <StyledChainImage
+          variant={variant}
+          src={chainLogoSrc}
+          width={Math.floor(width / 3)}
```

Comment:
why divide 3?

Reply:
from the design size

Reply:
the chain logo is about 1/3 of the total width